starknetup.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/sh
```

Comment:
why have you removed that?

Reply:
I initially removed it while attempting to source asdf to run asdf update and assumed using the user-preferred shell would suffice. 

On the second thought though, to ensure predictable behavior across all shells, I suppose it’s better to leave the shebang. Will revert.